backend/generator.js

```diff
@@ -10,10 +10,10 @@ exports.executeScript = function (socket, formData) {
     return false;
   }
   var email = formData.email;
-  var gitUrl = formData.gitUrl;
+  var gitUrl = "https://github.com/" + formData.username + "/" + formData.reponame + ".git";
```

Comment:
Remove gitUrl entirely. instead pass reponame and username to the script and build the url there